README.md

```diff
@@ -34,7 +34,7 @@ import ChipInput from 'material-ui-chip-input'
 | Name | Type | Default | Description |
 | --- | --- | --- | --- |
 | chipRenderer | `function` | | A function of the type `({ value, text, isFocused, isDisabled, handleClick, handleRequestDelete }, key) => node` that returns a chip based on the given properties. This can be used to customize chip styles. |
-| clearOnBlur | `bool` | `true` | Clears the input value after the component looses focus if set to true. |
+| clearOnBlur | `bool` | `true` | If true, clears the input value after the component loses focus. |
```

Comment:
Did this because I believe it reads better. Maybe should have restrained myself :upside_down_face: